addons/stock/models/stock_orderpoint.py

```diff
@@ -185,6 +185,10 @@ def write(self, vals):
             for orderpoint in self:
                 if orderpoint.company_id.id != vals['company_id']:
                     raise UserError(_("Changing the company of this record is forbidden at this point, you should rather archive it and create a new one."))
+        if 'snoozed_until' in vals:
+            for orderpoint in self:
+                if orderpoint.trigger == 'auto':
```

Comment:
```suggestion
            if any(o.trigger == 'auto' for o in self):
```

Reply:
You should also block the create no?

Reply:
You are probably right I didn't think of that use case since the `snooze_until` field is not part of the orderpoint views.